templates/OperationResponseExceptionSnippet.stencil

```diff
@@ -1,14 +1,20 @@
 {% for exception in op.exceptions %}
-    {% if exception.hasDefaultException == false %}
     if [
     {% for statusCode in exception.statusCodes %}
       {{ statusCode }},
     {% endfor  %}
     ].contains(statusCode)
     {
         do {
-            let decoder = JSONDecoder()
-            let decoded = try decoder.decode({{ exception.objectType }}.self, from: data)
+            {% if exception.strategy == "intBody" %}
+                {% include "OperationResponseExceptionIntBodySnippet.stencil" %}
+            {% elif exception.strategy == "stringBody" %}
+                {% include "OperationResponseExceptionStringBodySnippet.stencil" %}
+            {% elif exception.strategy == "jsonBody" %}
+                {% include "OperationResponseExceptionJsonBodySnippet.stencil" %}
+            {% else %}
```

Comment:
These snippets are so small I'd be okay with them being inline here. `OperationResponseExceptionIntBodySnippet` doesn't aid my comprehension, particularly if this is the only place it's used.

Reply:
We can aid the readability and maintainability of this by using stencil comments
`{# ... #} for comments not included in the template output`

Reply:
Fixed.